core/profiles/standard/standard.install

```diff
@@ -440,17 +434,25 @@ function standard_install() {
   // Update the menu router information.
   menu_rebuild();
 
-  // Enable the admin theme.
-  db_update('system')
-    ->fields(array('status' => 1))
-    ->condition('type', 'theme')
-    ->condition('name', 'seven')
-    ->execute();
-  config_set('system.core', 'admin_theme', 'seven');
-  config_set('system.core', 'node_admin_theme', '1');
+  $default_theme = 'basis';
+  $admin_theme = 'seven';
+  // Enable the Basis and Seven themes, and disable the Stark theme.
+  theme_enable(array($default_theme, $admin_theme));
+  theme_disable(array('stark'));
+  config('system.core')
+    // Set the Basis theme as the default theme.
+    // @see system_install()
+    ->set('theme_default', $default_theme)
+    // Set the Seven theme as the admin theme.
+    ->set('admin_theme', $admin_theme)
+    ->set('admin_theme_same_as_default', 0)
+    ->set('node_admin_theme', 1)
+    ->save();
 
   // Enable the page cache and aggregation.
-  config_set('system.core', 'preprocess_css', '1');
-  config_set('system.core', 'preprocess_js', '1');
-  config_set('system.core', 'cache', '1');
+  config('system.core')
```

Comment:
Since these are also variables in the same config file, we don't need to load the file again.
```php
$config->set('preprocess_css', 1);
$config->set('preprocess_js', 1);
$config->set('cache', 1);

$config->save();
```

Reply:
...and finally done 🙂